src/applications/disability-benefits/all-claims/submit-transformer.js

```diff
@@ -262,6 +262,27 @@ export const stringifyRelatedDisabilities = formData => {
   return clonedData;
 };
 
+export const cleanUpPersonsInvolved = incident => {
+  // We don't want to require any PTSD entries, but EVSS is rejecting any
+  // personsInvolved that don't have a "injuryDeath" type set. So we're
+  // setting blank entries to "other" and adding a generic other description
+  // see https://github.com/department-of-veterans-affairs/va.gov-team/issues/21422
+  const personsInvolved = incident.personsInvolved?.map(
+    person =>
+      person.injuryDeath
+        ? person
+        : {
+            ...person,
+            injuryDeath: 'other',
+            injuryDeathOther: person.injuryDeathOther || 'Entry left blank',
+          },
+  );
+  return {
+    ...incident,
+    personsInvolved,
+  };
+};
+
```

Comment:
This may need to be modified to include a name object and/or description per [this comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/21422#issuecomment-803066149).